agent/config/types.go

```diff
@@ -98,6 +99,10 @@ type Config struct {
 	// AppArmorCapable specifies whether the Agent is capable of using AppArmor
 	// security options
 	AppArmorCapable bool
+
+	// TaskCleanupWaitDuration specifies the time to wait after a task is stopped
+	// until cleanup of task resources is started.
+	TaskCleanupWaitDuration time.Duration
```

Comment:
I'm wondering if "task cleanup" implies image cleanup as well, which this does not handle.  What are your thoughts?